packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -125,13 +125,6 @@ class PipelineController extends CanvasController {
       pipelineId: item.pipelineId,
     };
 
-    const nodeDef = this.getAllPaletteNodes().find((n) => n.op === item.op);
```

Comment:
I don’t think we should delete this

Reply:
This can be removed because it's calling editActionHandler with editType: "createNode", so the code added to handleEditAction in this PR is going to be run anyway.